docs/dependency-updates.md

```diff
@@ -0,0 +1,164 @@
+which periodically checks for new versions of dependencies, and then submits
+pull requests to update them.
+
+Some terms to be aware of:
+
+- **Checks**: at the bottom of a Pull Request's page, GitHub will list the
+  outcomes of Continuous Integration runs. It calls these "checks".
+- **CI results**: to view the results of a CI run (e.g. `test_frontend`), click
+  the "Details" link next to the respective Check.
+- **Preview deployment**: for
+  every PR, a copy of the frontend with that PR's code is deployed to Netlify. You
+  can view that deployment by clicking the "Details" link next to the
+  `netlify/fx-relay-demo/deploy-preview` Check.
+
+All dependency updates involve verifying that all Checks are successful,
+investigating the results of failing CI runs, and a cursory inspection of the
+changelog (if any) for anything that might be relevant to Relay, in particular
+if a dependency's [major version](https://semver.org/) is bumped, or if a
+dependency doesn't use Semantic Versioning.
+
+If you're unsure whether anything looks as expected, try to replicate what
+you're doing on the `main` branch. If the results differ, there's probably an
+issue with the update.
+**Make sure to do a fresh install of the dependencies when switching branches!**
+
+You can line up multiple Dependabot PRs for merging by adding the comment
+`@dependabot merge`. Dependabot will then automatically keep the PR up-to-date
+with changes in `main`, and merge it if the Checks continue to run successfully.
+Alternatively, you can use the CLI tool [`pmac`](https://github.com/willkg/paul-mclendahand).
+
+## npm
+
+We have two npm projects in this repository: the website frontend (identified by
+`/frontend/package.json`), and the root project from which our end-to-end tests
+are run (identified by `/package.json`). The former is defined as a
+[workspace](https://docs.npmjs.com/cli/v8/using-npm/workspaces) in the latter,
+and both have their dependencies checked by Dependabot.
+
+What follows is a list of dependencies and how to check for potential breakage when they release new versions:
+
+### `*lint*`, `@types/*`, `typescript`, `*jest*`, `@testing-library/*`, `react-test-renderer`, `fast-check`, `prettier`
+
+For linters, type definitions, and non-Playwright testing-related packages, a
+successful CI run generally provides enough confidence that the upgrade is fine.
+That said, it can't harm to occassionally scan the results of the
+`build_frontend` or `test_frontend` CI runs to see if any new warnings were
+added. If `jest-junit` is updated, you'll want to check that CircleCI can still
+see code coverage for the frontend.
+
+### `@playwright/test` and `dotenv`
+
+Our Playwright tests are currently only triggered in CI once a day from the
+`main` branch. Thus, there's unfortunately no easy way to verify that it still
+runs successfully until after merging. To do so, after merging the PR, open the
```

Comment:
I was going back-and-forth on this since I believe it's still pretty hard to get a consistent result with the current tests between the local environment and the CI, but I guess it's unlikely to succeed locally even though it would fail in CI, so added in d9acd6a5.